etna/core/templatetags/sections_tags.py

```diff
@@ -20,3 +25,20 @@ def jumplinks(page):
     return {
         "sections": sections,
     }
+
+
+@register.filter
+def cookie_usage(record: dict) -> bool:
+    """
+    Return the True/False based on cookie usage value
+    if no cookie set it will return False
+    """
+    try:
+        usage = False
+        if "cookies_policy" in record:
+            cookie_str = unquote(record["cookies_policy"])
+            # Use Record property if available
+            usage = json.loads(cookie_str)["usage"]
+    except JSONDecodeError:
+        usage = False
+    return usage
```

Comment:
I'm not sure `section_tags.py` is the right place for this particular functionality. I would suggest adding a new module for this, e.g. `cookie_tags.py`